test/test-utils/src/main/java/software/amazon/awssdk/testutils/LogCaptor.java

```diff
@@ -52,13 +56,16 @@
 
 public interface LogCaptor extends SdkAutoCloseable {
```

Comment:
Do we have unit tests that cover this class?

Reply:
We have other tests depending on it that I used to test the new behavior, but let me add some dedicated unit tests as well.